src/richie/apps/courses/defaults.py

```diff
@@ -89,7 +89,7 @@
         "can_publish": False,
         "can_change_permissions": False,
         "can_move_page": True,
-        "can_view": True,
+        "can_view": True,  # beware: can_view = True makes it a view restriction...
```

Comment:
So this means that course page access is restricted; anonymous users can search and filter courses from the courses page view, but cannot click on a page to get a detailed view of a course?

Reply:
sorry that's a typo... these defaults are not tested by my test because the test includes a mock. I fixed it.